routes/index.js

```diff
@@ -7,4 +7,8 @@ router.get('/', function (req, res, next) {
   res.render('index', { products: products });
 });
 
+router.get('/fiveisprime', function (req, res) {
+  res.send('fiveisprime');
```

Comment:
Interesting choice for a response. I would have gone with "ONE OF US, ONE OF US".